oktoberfest/runner.py

```diff
@@ -79,7 +80,11 @@ def _preprocess(spectra_files: List[Path], config: Config) -> List[Path]:
         logger.info(f"Read {len(search_results)} PSMs from {internal_search_file}")
 
         # filter search results
-        search_results = pp.filter_peptides_for_model(peptides=search_results, model=config.models["intensity"])
+        if config.predict_intensity_locally:
+            model_type = "prosit"
```

Comment:
not sure if this is always correct. The idea is that users can train their own models with dlomix, so it isn't prosit.